src/main/java/net/minecraftforge/client/event/ClientPauseEvent.java

```diff
@@ -20,7 +20,12 @@
  *
  * <p>This event is fired on the {@linkplain MinecraftForge#EVENT_BUS main Forge event bus},
  * only on the {@linkplain LogicalSide#CLIENT logical client}.</p>
+ *
+ * @deprecated use {@linkplain ClientPauseChangeEvent.Post ClientPauseChangeEvent.Post} instead
+ * @see ClientPauseChangeEvent.Pre
+ * @see ClientPauseChangeEvent.Post
  */
+@Deprecated(forRemoval = true)
```

Comment:
This looks fine, just extra event being fired for the 1.20.4 lifecycle but not a big deal. However typically we add a since="1.20.4" to the deprecated annotation so that the build script will error out if we forget to remove it in 1.20.5.

Reply:
done